dark/assets/css/styles.css

```diff
@@ -1122,4 +1129,4 @@ html {
   border-radius: 2rem;
   font-size: 1.1rem;
   transition: 0.2 ease;
-}
+}
```

Comment:
Add end of the line @priyankarpal 
```suggestion
}

```

Reply:
Do these above mentioned changes @priyankarpal